hermes/bin/src/runtime_extensions/app_config.rs

```diff
@@ -0,0 +1,29 @@
+use crate::app::HermesAppName;
+
+/// ! Hermes application configuration for modules.
+
+/// Configuration struct for `SQLite` database.
+///
+/// This struct holds configuration options for `SQLite` database, including the path to
+/// the database file and the maximum size of the database.
+
+pub(crate) struct SqliteConfig {
+    /// Path to the `SQLite` database file, not set if it's in-memory database.
+    pub(crate) db_file: Option<String>,
+    /// Maximum size of the `SQLite` database in bytes.
+    pub(crate) max_db_size: u32,
+}
+
+pub(crate) fn get_app_persistent_sqlite_db_cfg(_app_name: HermesAppName) -> Option<SqliteConfig> {
```

Comment:
This one, because we will return config according into the inputed `app_name`. Right now, it's just dummy to make things works. So when the config can't be found from the given `app_name`, it should return with `None`.